### PR TITLE
Fix param.Action (#9).

### PR DESCRIPTION
**Pull request by ceball**

### CODE REVIEW

File: parambokeh/__init__.py [hunk 1]
```diff
@@ -174,6 +174,9 @@ def __call__(self, parameterized, doc=None, plots=[], **params):
     def on_msg(self, msg):
         p_name = msg['p_name']
         p_obj = self.parameterized.params(p_name)
+        if isinstance(p_obj, param.Action):
+            getattr(self.parameterized, p_name)(self.parameterized)
+            return
```
Comment (ceball) [inline parambokeh/__init__.py:179]: Other widgets won't be refreshed - is that a problem? What if the callback alters another parameter value? (I decided not to get into this now, deferring it to 'dependencies between parameters and code'...)

Comment (philippjfr) [reply parambokeh/__init__.py:179]: >deferring it to 'dependencies between parameters and code'

+1

File: parambokeh/comms.py [hunk 2]
```diff
@@ -225,6 +225,10 @@ def _handle_msg(self, msg):
                 with StandardOutput() as stdout:
                     self._on_msg(msg)
         except Exception as e:
+            # TODO: isn't this cutting out info needed to understand what's gone wrong?
+            # Since it's only going to the js console, maybe we could just show everything
+            # (error = traceback.format_exc() or something like that)? Separately we do need a mechanism
+            # to report reasonable messages to users, though.
```
Comment (philippjfr) [inline parambokeh/comms.py:231]: All true, if we can find a way to make tracebacks readable on the JS console I'd be very happy to change this (and port the change back to holoviews).

File: parambokeh/__init__.py (hunk 1 above)
Comment (jbednar) [reply parambokeh/__init__.py:179]: Sounds fine.

File: examples/user_guide/Introduction.ipynb [hunk 3]
```diff
@@ -124,6 +124,22 @@
     "Example.num_int"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Example.timestamps records the times you pressed the \"record timestamp\" button."
```
Comment (jbednar) [inline examples/user_guide/Introduction.ipynb:131]: Good example, thanks.

**Comment by jbednar** [comment]
Happy to merge, but it would be nice if conda was working so the tests could complete.

**Comment by ceball** [comment]
Seems like the same problem reported in https://github.com/conda/conda/issues/6468. However, the version of conda has not changed since the last successful build (4.3.30), so I'd guess something has changed at travis (e.g. related to docker, or the linux image, etc). I have not investigated.

**Comment by ceball** [comment]
The problem appears to be intermittent :(

E.g. two jobs in the same build (same codebase, same image); one gets past the conda download/install stage, the other one fails: https://travis-ci.org/ioam/parambokeh/builds/317743629?utm_source=github_status&utm_medium=notification

**Pull request merged**